portfolio/src/main/webapp/index.html

```diff
@@ -20,22 +20,29 @@ <h2>A bit about me</h2>
     <ul>Guilty pleasure: binge-watch The Big Bang Theory</ul>
     <ul>Fun fact: ❤raw carrots ⛔cooked carrots</ul>
     <br>
-    <h2>Where am I?</h2>
-    <button onclick="findZoe()">Find Zoe</button>
+    <h2>Where am I? <button onclick="findZoe()">Find Zoe</button></h2>
     <div id="link-container">
       <a id="link"></a>
     </div>
     <br>
-    <h2>Help me <button onclick="grow()"><b>Grow</b></button></h2>
+    <h2>Help me <button onclick="grow()">Grow</button></h2>
     <img id="basil" />
     <p id="endMsg"></p>
     <br>
     <h2>Comments</h2>
     <form action="/data" method="POST">
```

Comment:
Here is how to fix the issue where you post a comment and the limit goes back to default

- Remove action and method attribute from form element so it doesn't redirect the user to the /data page (which your servlet then forces the user to be redirected back to the main page)
- Create a submitComment method and attach it to the submit button method
- In the submitComment method use fetch(...) method to make an async post request to /data to submit the comment (search online on how to use fetch to make post request with parameters)
- In your doPost remove the response.sendRedirect call. This is what is making the limit to go back to its default because its just reloading the page.
- Since now you aren't forcing the user to refresh the page, your onload method that calls getComments won't be called which means the new comments won't show up. To show the comments, after you make the fetch call in the submitComment, just call your getComments() method

I hope this makes sense. If something doesn't make sense, please ask!

Reply:
btw this would be something nice to present in our pod sometime next week.

Reply:
Thanks for the input! The way you suggested resolved the issue perfectly. I have made the changes.